test/unit/streaming-client.spec.ts

```diff
@@ -26,7 +26,13 @@ describe('streaming-client', () => {
     describe('start', () => {
         it('Connects to api with parameters', () => {
             // Arrange
-            const config = new SessionConfig('my metadata', 'my custom vocab id', true, true);
+            const config = new SessionConfig(
+                'my metadata', 
+                'my custom vocab id', 
+                true, 
+                true,
+                0
```

Comment:
Could we change these to named parameters? It will be easier to figure out which arg belongs to which param.

Reply:
This is sadly not possible.